src/components/Cluster.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const MIN_CLUSTER_DIAMETER = 30
+
+const MAX_CLUSTER_DIAMETER = 100
+
+// TODO: Cluster styling/icon that Siraj wants
+const ClusterContainer = styled.button`
+  width: ${(props) => props.diameter}px;
+  height: ${(props) => props.diameter}px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+  background: rgba(35, 74, 201, 0.214);
+  transform: translate(-50%, -50%);
+`
+
+const Cluster = (props) => {
+  const { onClick, lat, lng, count } = props
```

Comment:
I think it's cleaner to unpack all props inside the function parameter, e.g.

`({ onClick, lat, lng count }) => {`